6_Loops.ipynb

```diff
@@ -9,7 +9,7 @@
     "\n",
     "## 6.1 Introduction\n",
     "\n",
-    "Another important feature of computer programs is that they can do the same thing over and over again with different information. To allow this, it is possible to use loops in the program; essentially a loop is executed until it runs out of data or the code decides to break out of it.\n",
+    "Another important feature of computer programs is that they can do the same thing over and over again with different information. To allow this, it is possible to use loops in the program; essentially a loop until a given condition is met.\n",
```

Comment:
Another important feature of computer programs is that they can do the same thing over and over again with different information. To allow this, it is possible to use loops in the program; essentially a loop is executed until a given condition is met.